### ISSUE TITLE
How to set/make sure the batchnorm is in eval mode in tf.keras?

### ISSUE THREAD

**Issue by WeiFoo** [WeiFoo/keras_pytorch#1]
In this simpleconvnet, I've used a batchnorm layer, but not quite sure how to set it to eval mode. I found that in the API, https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization

In the call arguments, there's a `training` parameter to control this. However, if I save and reload the model, the momentum, beta... those parameters are not properly loaded(I guess). Then I'm not sure whether the batchnorm is working as expected in eval mode.
